pom.xml

```diff
@@ -587,6 +614,22 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>${codehause.license.plugin.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>com.hazelcast.maven</groupId>
+                    <artifactId>attribution-maven-plugin</artifactId>
+                    <version>1.0</version>
+                    <configuration>
+                        <exclusionPatterns>
+                            <exclusionPattern>^Copyright Laws of the United States.</exclusionPattern>
+                            <exclusionPattern>per-property override</exclusionPattern>
+                            <exclusionPattern>^copyright notice</exclusionPattern>
+                            <exclusionPattern>FAILED_PRECONDITION</exclusionPattern>
+                            <exclusionPattern>The ASF licenses this file to you under the Apache License</exclusionPattern>
+                            <exclusionPattern>Substitute for l, l</exclusionPattern>
+                            <exclusionPattern>inside edge crossings that are between</exclusionPattern>
+                        </exclusionPatterns>
```

Comment:
Maybe some explanation for this would be good.

Reply:
:+1: I'll add an explanation.